frontend/src/components/CookieBanner/index.tsx

```diff
@@ -22,32 +25,30 @@ const CookieBanner: FC = () => {
     setIsHidden(true);
   }
 
-  if (!isHidden) {
-      return (
-          <Wrapper>
-              By using this site, you are agreeing to our{" "}
-              <Link href={TOS_LINK} target="_blank" rel="noreferrer">
-                  terms of service
-              </Link>
-              . We use cookies to help us improve our future efforts, and we also use
-              necessary cookies to make our site work. To learn more, read our{" "}
-              <Link href={PRIVACY_LINK} target="_blank" rel="noreferrer">
-                  privacy policy
-              </Link>
-              .
-              <ButtonWrapper>
-                  <OKButton onClick={handleOKClick}>I&lsquo;m OK with cookies</OKButton>
-                  <NoButton onClick={handleNoClick}>No thanks</NoButton>
-              </ButtonWrapper>
-          </Wrapper>
-      );
-  }
-
-  return null;
-
+  return (
+    <Wrapper>
+      By using this site, you are agreeing to our{" "}
+      <Link href={TOS_LINK} target="_blank" rel="noreferrer">
+        terms of service
+      </Link>
+      . We use cookies to help us improve our future efforts, and we also use
+      necessary cookies to make our site work. To learn more, read our{" "}
+      <Link href={PRIVACY_LINK} target="_blank" rel="noreferrer">
+        privacy policy
+      </Link>
+      .
+      <ButtonWrapper>
+        <OKButton onClick={handleOKClick}>I&lsquo;m OK with cookies</OKButton>
+        <NoButton onClick={handleNoClick}>No thanks</NoButton>
+      </ButtonWrapper>
+    </Wrapper>
+  );
 };
 
 function hasClickedOnBanner() {
+  // (thuang): We don't wanna show the banner on the SSR'd page
+  if (isSSR()) return true;
```

Comment:
In order for SSR'd page to not show cookie banner, we need to return `true` here